validator/block_validator.go

```diff
@@ -500,6 +514,23 @@ func (v *BlockValidator) sendValidations(ctx context.Context) {
 			v.ProcessBatches(v.globalPosNextSend.BatchNumber, [][]byte{seqMsg})
 			seqBatchEntry = seqMsg
 		}
+		v.blockMutex.Lock()
+		if v.lastBlockValidatedUnknown {
+			firstMsgInBatch := arbutil.MessageIndex(0)
+			if v.globalPosNextSend.BatchNumber > 0 {
+				var err error
+				firstMsgInBatch, err = v.inboxTracker.GetBatchMessageCount(v.globalPosNextSend.BatchNumber - 1)
+				if err != nil {
+					log.Error("validator couldnt read message count", "err", err)
+					return
```

Comment:
`v.blockMutex` needs released here

Reply:
We should also probably log an info line not an error line if the error is MissingAccumulator, meaning that we're still just waiting to catch up